doc/ipython-notebooks/gaussian_process/variational_classifier.ipynb

```diff
@@ -629,14 +733,380 @@
      "level": 2,
      "metadata": {},
      "source": [
-      "A Real-World Example"
+      "<a id='visu'>An Example for Visualization</a> <a href='#toc'>[TOP]</a>"
      ]
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "We apply variational methods to the sonar data set. The data set can be found at <a href=\"https://archive.ics.uci.edu/ml/machine-learning-databases/undocumented/connectionist-bench/sonar/\">here</a>. This is a binary classification problem. The goal of this example is to compare all inference methods implemented in shogun in term of making prediction predictions.  The following is the description of the data set. This example is about reproducing the example at page 29 of <a href=\"http://www.jmlr.org/papers/volume9/nickisch08a/nickisch08a.pdf\">this paper</a>. Please refer the paper about the definition of the information bit measure.\n",
+      "We use the banana data set to demonstrate decision boundary of GPC. The data set can be found at <a href=\"http://sci2s.ugr.es/keel/dataset.php?cod=182\">here</a>. This is a binary classification problem. The goal of this example is to visually show the decision boundary of GPC models. The following is the description of the data set.\n",
+      "\n",
+      "An artificial data set where instances belongs to several clusters with a banana shape. There are two attributes At1 and At2 corresponding to the x and y axis, respectively. The class label (-1 and 1) represents one of the two banana shapes in the dataset."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "def train(inference, linesearch, likelihood, x_train, y_train, kernel_log_sigma, kernel_log_scale):\n",
```

Comment:
Before the code start. Could you add a sentence on what type of things you are going to do?
Something as "We now define code to train a GP classifier using Shogun."